packages/nodes-base/utils/sendAndWait/utils.ts

```diff
@@ -357,6 +357,12 @@ export async function sendAndWaitWebhook(this: IWebhookFunctions) {
 	const res = this.getResponseObject();
 	const req = this.getRequestObject();
 
+	const isTokenValid = this.validateExecutionWaitingToken();
+
+	if (!isTokenValid) {
+		throw new Error('Invalid waiting token');
```

Comment:
Should this be a `NodeOperationError` or another class?
Could `validateExecutionWaitingToken` do the throw?